code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/AlekseevSergeiTest.java

```diff
@@ -0,0 +1,78 @@
+package ru.unn.agile.ComplexNumber.model;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class AlekseevSergeiTest {
+
+    @Test
+    public void areAddingZeroDoesntChangeNumber() {
+        ComplexNumber z = new ComplexNumber(1.1, 2.3);
+        ComplexNumber zero = new ComplexNumber(0.0, 0.0);
+
+        ComplexNumber sum = z.add(zero);
+
+        assertTrue(sum.equals(z));
+    }
+
+    @Test
+    public void areMultiplicationByZeroChangeNumber() {
+        ComplexNumber z = new ComplexNumber(1.1, 2.3);
+        ComplexNumber zero = new ComplexNumber(0.0, 0.0);
+
+        ComplexNumber mult = z.multiply(zero);
+
+        assertTrue(mult.equals(zero));
```

Comment:
similar here(not to equals in assert):

    public void areMultiplicationByZeroChangeNumber() {
        //Arrange
        ComplexNumber z = new ComplexNumber(1.1, 2.3);
        ComplexNumber zero = new ComplexNumber(0.0, 0.0);

        //Act
        ComplexNumber mult = z.multiply(zero);

        //Assert
        assertEquals(mult, zero);